firefox_privacy_notice/sr.md

```diff
@@ -1,71 +1,71 @@
 # Obaveštenje o privatnosti pretraživača Firefox
 
-15\. april 2014.
-{: datetime="2014-04-15" }
+15. april 2014.
+{: datetime=2014-04-15 }
```

Comment:
Not sure about removing the quotes around the datatime value. I did notice that other files used them still, ie: `{: datetime="2014-08-29" }`